prereise/gather/HIFLD_Profile_APP/Demand/demand_divide.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
```

Comment:
The above lines are unnecessary. Start with the import statement

Reply:
I guess Mingze used some template for every new script... @Mingze-Li128 please remove all the unnecessary comments